site/src/analytics-integrations.ts

```diff
@@ -22,6 +22,11 @@ const DEFAULT_DOMAINS: string[] = [
   "rpi_power",
 ];
 
+// Integrations without integration documentation pages
+const DOMAINS_WITHOUT_DETAILS: Record<string, IntegrationDetails> = {
+  energy: { title: "Energy", quality_scale: "internal" },
```

Comment:
Should we add a quick page instead?

Reply:
We did not want that?
Several PR's have been closed trying to add that.